internal/fsutil.go

```diff
@@ -72,6 +73,44 @@ func createTestExtImg(ctx context.Context, t *testing.T, extName string, testFil
 	return imgFile.Name()
 }
 
+// Devicefile represents a block device
+type Devicefile struct {
+	Subpath string
+	UID     int
+	GID     int
+	Dev     int // major number
+}
+
+// CreateBlockDevice creates a block device, or block special file for testing
+func CreateBlockDevice(ctx context.Context, t *testing.T, extName string, testFile Devicefile) string {
+	t.Helper()
+	tempdir, err := ioutil.TempDir("", "")
```

Comment:
The tempdir wouldn't be automatically deleted.

Reply:
It contains the device file which will be used to bind mount later, we can't remove it at this point. A tempdir created by [createTestExtImg](https://github.com/firecracker-microvm/firecracker-containerd/blob/c2323bc71886b3abfefd9afa53244740f70db0b8/internal/fsutil.go#L51) didn't remove its tempdir as well.

Reply:
But with the update code, an error was returned, is that why we need to remove tempdie in order to resolve this?
```
/tmp/With_Jailer_and_bind-mount509265757/dev/sda is apparently in use by the system; will not make a filesystem here!
```

Reply:
I just don't want to ask developers to remove /tmp directory regularly just because we don't remove files.

Not really sure the error you got. Did you mount the file in somewhere?

Reply:
Nope. I didn't mount the file. It is a new file generated every time CreateBlockDevice() is called, given the fact that we are creating the block device file under tempDir.